src/libs/DateUtils.js

```diff
@@ -291,6 +291,104 @@ function getDateStringFromISOTimestamp(isoTimestamp) {
     return dateString;
 }
 
+/**
+ * @returns {string} example: 2023-05-16 05:34:14
+ */
+function getThirtyMinutesFromNow() {
+    return moment().add(30, 'minutes').format('YYYY-MM-DD HH:mm:ss');
+}
+
+/**
+ * @returns {string} example: 2023-05-16 05:34:14
+ */
+function getOneHourFromNow() {
+    return moment().add(1, 'hours').format('YYYY-MM-DD HH:mm:ss');
+}
+
+/**
+ * @returns {string} example: 2023-05-16 05:34:14
+ */
+function getEndOfToday() {
+    return moment().endOf('day').format('YYYY-MM-DD HH:mm:ss');
+}
+
+/**
+ * @returns {string} example: 2023-05-16 05:34:14
+ */
+function getEndOfWeekFromNow() {
+    return moment().add(7, 'days').format('YYYY-MM-DD HH:mm:ss');
+}
+
+/**
+ * @param {string} time
+ * @returns {string} example: 2023-05-16
+ */
+function extractDate(time) {
+    return moment(time).format('YYYY-MM-DD');
+}
+
+/**
+ * @param {string} dateTimeString
+ * @returns {string} example: 11:10
+ */
+function extractTime(dateTimeString) {
```

Comment:
Unsure about this, but it might make sense to use `extractTime24Hour` here so we don't encounter problems in the future where the format is assumed to be 12 hour.

Reply:
👍🏻